includes/wp-cli/class-rest-api.php

```diff
@@ -86,24 +88,22 @@ public function get_queue( $args, $assoc_args ) {
 	private function format_events( $events ) {
```

Comment:
This is...a bit weird. It's a CLI class that runs the REST endpoints. Honestly thinking we can probably 'nix this whole part later on.